ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_logic.py

```diff
@@ -1143,3 +1143,68 @@ def test_jax_numpy_packbits(
         axis=axis,
         bitorder=bitorder,
     )
+
+
+@st.composite
+def _func_and_shape_dtype_helper(draw):
+    # here assumption is that the input func will take the len(shape) no of parameters
+    def add_numbers(*args):
+        total = 0
+        for num in args:
+            total += num
+        return total
+
+    shape = draw(
+        helpers.get_shape(
+            allow_none=False,
+            min_num_dims=1,
+            max_num_dims=3,
+            min_dim_size=1,
+            max_dim_size=3,
+        )
+    )
+
+    dtype = draw(
+        st.sampled_from(
+            [
+                "float16",
+                "float64",
+                "int8",
+                "int16",
+                "float32",
+                "int32",
+                "int64",
+            ]
+        )
+    )
```

Comment:
I guess you should use the `helpers.get_dtypes("valid")` to generate valid dtypes for this rather than hardcoding these here